opencolorio_config_aces/clf/transforms/utilities.py

```diff
@@ -0,0 +1,175 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+"""
+CLF Generation Utilities
+========================
+
+Defines various utility functions for generating *Common LUT Format*
+(CLF) transforms.
+"""
+
+import PyOpenColorIO as ocio
+
+from opencolorio_config_aces.utilities import required
+
+__author__ = "OpenColorIO Contributors"
+__copyright__ = "Copyright Contributors to the OpenColorIO Project."
+__license__ = "New BSD License - https://opensource.org/licenses/BSD-3-Clause"
```

Comment:
I know this is already in a lot of the other modules and totally unrelated to this PR, but when we switched OCIO over to using SPDX we removed a lot of the earlier license/copyright notices.  Just wondering if it is following the best practice to repeat what is in the SPDX above?

Reply:
They are useful because available through introspection which the SPDX comment is not, e.g. `python -c "import opencolorio_config_aces;help(opencolorio_config_aces.config.cg.generate.config)"`

```
Help on module opencolorio_config_aces.config.cg.generate.config in opencolorio_config_aces.config.cg.generate:

NAME
    opencolorio_config_aces.config.cg.generate.config

DESCRIPTION
    *ACES* Computer Graphics (CG) Config Generator
    ==============================================
    
    Defines various objects related to the generation of the *ACES* Computer
    Graphics (CG) *OpenColorIO* config:
    
    -   :func:`opencolorio_config_aces.generate_config_cg`

FUNCTIONS
    clf_transform_to_colorspace(clf_transform, describe=<ColorspaceDescriptionStyle.LONG_UNION: 22>, signature_only=False, **kwargs)
        Generate the *OpenColorIO* colorspace for given *CLF* transform.
        
        Parameters
        ----------
        clf_transform : CLFTransform
            *CLF* transform.
        describe : bool, optional
            Whether to use the full *CLF* transform description or just its ID.
        signature_only : bool, optional
            Whether to return the *OpenColorIO* colorspace signature only, i.e. the
            arguments for its instantiation.
        
        Other Parameters
        ----------------
        \**kwargs : dict, optional
            Keywords arguments for the
            :func:`opencolorio_config_aces.colorspace_factory` definition.
        
        Returns
        -------
        Object
            *OpenColorIO* colorspace.
    
    clf_transform_to_description(clf_transform, describe=<ColorspaceDescriptionStyle.LONG_UNION: 22>)
        Generate the *OpenColorIO* colorspace or named transform description for
        given *CLF* transform.
        
        Parameters
        ----------
        clf_transform : CLFTransform
            *CLF* transform.
        describe : bool, optional
            Whether to use the full *CLF* transform description  or just the
            first line.
        
        Returns
        -------
        unicode
            *OpenColorIO* colorspace or named transform description.
    
    generate_config_cg(data=None, config_name=None, validate=True, describe=<ColorspaceDescriptionStyle.SHORT_UNION: 14>, config_mapping_file_path=PosixPath('/Users/kelsolaar/Documents/Development/opencolorio/OpenColorIO-Config-ACES/opencolorio_config_aces/config/cg/generate/resources/OpenColorIO-Config-ACES _CG_ Transforms - CG Config - Mapping.csv'), additional_data=False)
        Generate the *ACES* Computer Graphics (CG) *OpenColorIO* config.
        
        Parameters
        ----------
        data : ConfigData, optional
            *OpenColorIO* config data to derive the config from, the default is to
            use the *aces-dev* reference implementation *OpenColorIO* config.
        config_name : unicode, optional
            *OpenColorIO* config file name, if given the config will be written to
            disk.
        validate : bool, optional
            Whether to validate the config.
        describe : int, optional
            Any value from the
            :class:`opencolorio_config_aces.ColorspaceDescriptionStyle` enum.
        config_mapping_file_path : unicode, optional
            Path to the *CSV* mapping file used to describe the transforms mapping.
        additional_data : bool, optional
            Whether to return additional data.
        
        Returns
        -------
        Config or tuple
            *OpenColorIO* config or tuple of *OpenColorIO* config and
            :class:`opencolorio_config_aces.ConfigData` class instance.

DATA
    PATH_TRANSFORMS_MAPPING_FILE_CG = PosixPath('/Users/kelsolaar/Document...
    __all__ = ['PATH_TRANSFORMS_MAPPING_FILE_CG', 'clf_transform_to_descri...
    __copyright__ = 'Copyright Contributors to the OpenColorIO Project.'
    __email__ = 'ocio-dev@lists.aswf.io'
    __license__ = 'New BSD License - https://opensource.org/licenses/BSD-3...
    __maintainer__ = 'OpenColorIO Contributors'
    __status__ = 'Production'

AUTHOR
    OpenColorIO Contributors

FILE
    /Users/kelsolaar/Documents/Development/opencolorio/OpenColorIO-Config-ACES/opencolorio_config_aces/config/cg/generate/config.py
```